index.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+header('Content-Type: text/html; charset=UTF-8');
+
+require_once ("lib/Core.php");
+require_once ('vendor/autoload.php');
+
+$api = new SparkAPI_OIDCAuth("OAuth_key", "OAuth_secret", "OAuth_redirect_url");
+
+// identify your application (optional)
+$api->SetApplicationName("PHP-API-Code-Examples/1.0");
+// authenticate
+$api->Authenticate();
+
+// if we have a token, we will be ready to make requests
+$token = $api->GetAccessToken() ?: null;
+
+if (isset($token)) {
```

Comment:
I made it cleaner, but since I'm trying to do this as a one-page example, it resubmits to this page several times, so I needed a way to check if we're ready to make a request or if we're still in the token exchange dance.